spack/package.py

```diff
@@ -10,9 +10,9 @@ class Arbor(CMakePackage, CudaPackage):
     """Arbor is a high-performance library for computational neuroscience
     simulations."""
 
-    homepage = 'https://github.com/arbor-sim/arbor/'
+    homepage = 'https://arbor-sim.github.io/'
     git      = 'https://github.com/arbor-sim/arbor.git'
-    url      = 'https://arbor.readthedocs.io/en/latest/'
+    url      = 'https://github.com/arbor-sim/arbor/releases/download/v0.5/arbor-v0.5-full.tar.gz'
     maintainers = ['bcumming', 'halfflat']
```

Comment:
The A-team?

Reply:
This was based on a draft version that was written 18 months ago, so please don't be offended!
I will add you and @haampie, because he is a real "Spack A teamer"!

Reply:
I actually meant 'The A-team', literally. As in Arbor team ;)

Didn't @schmitts (co-)write this file?